anthos-bm-apigee/README.md

```diff
@@ -0,0 +1,81 @@
+
+## Apigee Hybrid on Anthos Baremetal on Google Compute Engine VMs with Terraform
+
+This repository is a sample Apigee Hybrid installation on Anthos Cluster on bare metal running on GCE Virtual Machine. This also assumes that Apigee organization name is same as the GCP Project Id. The installation will also create a trial Apigee Edge organization with same name as project id if  Apigee Organization is not available.
+
+### Pre-requisites
+
+- A workstation with access to internet _(i.e. Google Cloud APIs)_ with the following installed
+  - [Git](https://www.atlassian.com/git/tutorials/install-git)
+  - [Google Cloud SDK](https://cloud.google.com/sdk/docs/install)
+  - [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli) (>= v0.15.5, < v1.2)
+
+- A [Google Cloud Project](https://console.cloud.google.com/cloud-resource-manager?_ga=2.187862184.1029435410.1614837439-1338907320.1614299892) _(in which the resources for the setup will be provisioned)_
+
+- Enable Compute Engine API Services and Apigee API Services.
+
+- Ensure that the default Compute Engine developer account has Editor role. You can also create a new service account for the compute engine with Editor and specify the service account name in terraform variables.
+
+![Default Service Account](docs/images/default_service_account.png)
+
+- A [Service Account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) in the project that satisfies **one** of the following requirements and its **[key file downloaded](docs/create_sa_key.md)** to the workstation:
+    - The Service Account has `Owner` and `Apigee Organization Administrator` permissions
+    - The Service Account has both `Editor`, `Project IAM Admin` and `Apigee Organization Administrator` permissions
+The scripts to create service account and key creation is also mentioned in Quickstart module.
+- Organizational Policy Constraints
+      Follwowing list of Policy needs to be enabled for the organizations. If these are not enabled at Organizations, please consider them for the project.
```

Comment:
Line breaks in markdown don't show up verbatim on the rendered view. So we need to check for them. Usually it requires two spaces and then the line break. Can you please re-check the rendered view of the READMEs to ensure it has the expected formatting?

- The following list of **Organizational Policy Constraints** enabled on the Google Cloud Organization your project is on:
<TABLE HERE>

Reply:
Fixed